catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/view/Registry.view.js

```diff
@@ -272,21 +283,92 @@ define([
                     data = this.model.toJSON();
                 }
                 data.name = this.getNodeName();
+                data.remote = this.isRemote();
```

Comment:
Actually, because we are returning `data` in this function, should we then make this line of code read `data.remote = !this.mode.attributes.isLocal` ?